File: web3rpc/rpc-specs/code-samples/javascript/governance/itemsAt.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumber = 89;

    sdk.governance.itemsAt(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/governance/itemsAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = 0

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.items_at(blockTag)

print(governance_response)
